app/policies/user_policy.rb

```diff
@@ -0,0 +1,34 @@
+class UserPolicy 
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def show?
```

Comment:
set to true and move this logic to  a new show_photo? method to account for edge case. call show_photo? in users#show

Reply:
@MayaS1111 Where is your show_photo policy method?